examples/twittermap/web/public/javascripts/map/controllers.js

```diff
@@ -318,7 +317,30 @@ angular.module('cloudberry.map', ['leaflet-directive', 'cloudberry.common','clou
             });
         }
     };
-    
+
+    // load population Json to get state and county polygons, store in countmap cache.
+    $scope.loadPopJsonFiles = function loadPopJsonFiles(onEachFeature) {
```

Comment:
`onEachFeature` parameter is not necessary for this function, it was used for creating polygon objects by Leaflet.js library to register events listeners for `loadCityPolygons` function and etc.